provisioning/local/env-scripts/extract-variables-common.sh

```diff
@@ -5,15 +5,24 @@ cd "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 source ./functions.sh
 
 # output variables
-output_var 'TEST_TENANT_ID' $(terraform_output 'az_tenant_id')
-output_var 'TEST_CLIENT_ID' $(terraform_output 'az_application_id')
-output_var 'TEST_CLIENT_SECRET' $(terraform_output 'az_application_secret')
-output_var 'TEST_KEY_VAULT_URL' $(terraform_output 'az_key_vault_url')
+output_var 'TEST_TENANT_ID' "$(terraform_output 'az_tenant_id')"
+output_var 'TEST_CLIENT_ID' "$(terraform_output 'az_application_id')"
+output_var 'TEST_CLIENT_SECRET' "$(terraform_output 'az_application_secret')"
+output_var 'TEST_KEY_VAULT_URL' "$(terraform_output 'az_key_vault_url')"
 echo ""
 
-output_var 'TEST_AWS_ACCESS_KEY_ID' $(terraform_output 'aws_access_key_id')
-output_var 'TEST_AWS_SECRET_ACCESS_KEY' $(terraform_output 'aws_access_key_secret')
-output_var 'TEST_AWS_REGION' $(terraform_output 'aws_region')
-output_var 'TEST_AWS_KMS_KEY_ID' $(terraform_output 'aws_kms_key_id')
-output_var 'TEST_AWS_ROLE_ID' $(terraform_output 'aws_role_arn')
+output_var 'TEST_AWS_ACCESS_KEY_ID' "$(terraform_output 'aws_access_key_id')"
+output_var 'TEST_AWS_SECRET_ACCESS_KEY' "$(terraform_output 'aws_access_key_secret')"
+output_var 'TEST_AWS_REGION' "$(terraform_output 'aws_region')"
+output_var 'TEST_AWS_KMS_KEY_ID' "$(terraform_output 'aws_kms_key_id')"
+output_var 'TEST_AWS_ROLE_ID' "$(terraform_output 'aws_role_arn')"
+
+output_var 'TEST_GOOGLE_CLOUD_PROJECT' "$(terraform_output 'gcp_project_id')"
+output_var 'TEST_GCP_KMS_KEY_ID' "$(terraform_output 'gcp_kms_key_id')"
+PRIVATE_KEY_ENCODED="$(terraform_output 'gcp_private_key')"
+PRIVATE_KEY=$(printf "%s" "$PRIVATE_KEY_ENCODED" | base64 --decode --wrap=0)
+
+output_file 'gcp-private-key.json' "$PRIVATE_KEY"
```

Comment:
Jako ja bych sem narval rovnou `output_file 'gcp-private-key.json' "$(terraform_output 'gcp_private_key' | base64 --decode -w0)"`, ale mozna moc pospicham :smile:

Reply:
jj, až do okamžiku než se ti ve výstupu objeví 
![image](https://github.com/keboola/object-encryptor/assets/4319320/76474d37-c592-4911-b6e9-d7ee81a42c93)

:)

Reply:
To se ti povedlo jak? (tohle uz neberu jako soucast PR, spis ze zajimavosti protoze Bash)

Reply:
když gcp_private_key vrátí 'null' :)